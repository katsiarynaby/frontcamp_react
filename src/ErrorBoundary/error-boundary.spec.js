import React from 'react';
import { shallow } from 'enzyme';
import shallowToJson from 'enzyme-to-json';

import ErrorBoundary from './index';

describe('ErrorBoundary', () => {
  it("ErrorBoundary should render correctly", () => {
    const wrapper = shallow(<ErrorBoundary />);

    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});