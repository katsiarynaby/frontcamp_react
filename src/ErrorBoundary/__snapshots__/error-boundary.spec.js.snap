// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBoundary ErrorBoundary should render correctly 1`] = `""`;
